services/publicapi/results.go

```diff
@@ -35,13 +35,13 @@ func isOutputPotentiallyOutOfSync(config config.PublicApiConfig, referenceBlockT
 	return threshold > referenceBlockTimestamp
 }
 
-func validateRequest(config config.PublicApiConfig, protocolVersion primitives.ProtocolVersion, vcId primitives.VirtualChainId) (protocol.TransactionStatus, error) {
-	if primitives.ProtocolVersion(1) != protocolVersion {
+func validateRequest(cfg config.PublicApiConfig, protocolVersion primitives.ProtocolVersion, vcId primitives.VirtualChainId) (protocol.TransactionStatus, error) {
+	if protocolVersion > config.MAXIMAL_PROTOCOL_VERSION_SUPPORTED_VALUE{
```

Comment:
Please add a TODO, need to revisit the logic, thanks

Reply:
done